README.md

```diff
@@ -1,14 +1,33 @@
 # Maid
 Manages the cleaning of events and other things.
 
-##API
-	Maid.new()                        Returns a new Maid object.
+### API
+```cs
+Maid Maid.new()
```

Comment:
Uhm... my fault. Props to @Validark on the bulls-eye with the _better highlighting_ guess.